Doco review of header documentation (#4533)
Particular/docs.particular.net: nservicebus/messaging/headers.md

 - nservicebus/messaging/header-manipulation
---

> The headers of a message contains, similar to TTP headers, metadata about the message being sent over the queueing system. This document contains the information about the headers used by NSeriviceBus. To learn more about how to use custom headers, see the documentation on [manipulating message headers](/nservicebus/messaging/header-manipulation.md).

Reviewer: ```suggestion
The headers of a message are similar to HTTP headers and contain metadata about the message being sent over the queueing system. This document describes the headers used by NSeriviceBus. To learn more about how to use custom headers, see the documentation on [manipulating message headers](/nservicebus/messaging/header-manipulation.md).
```

Identifier of the conversation that this message is part of. It enables the tracking of message flows that span more than one message exchange. `Conversation Id` and `RelatedTo` fields allow [ServiceInsight](/serviceinsight/#flow-diagram) to reconstruct the entire message flow.

> The first message that is sent in a new flow is automatically assigned a unique `Conversation Id` that is then propagated to all the messages that are subsequently sent, thus forming a _conversation_. Each message that is sent within a conversation also has a `RelatedTo` value that identifies the message that caused it to be sent.

Reviewer: ```suggestion
The first message sent in a new flow is automatically assigned a unique `Conversation Id` that is then propagated to all the messages that are sent afterward, forming a _conversation_. Each message sent within a conversation has a `RelatedTo` value that identifies the message that caused it to be sent.
```

> In certain scenarios, Conversation Id has to be assigned manually because NServiceBus can't infer that messages are belong to the same conversation. For example, when a `CancelOrder` message needs to become part of an existing order conversation, then the Order Id can be used for as the Conversation Id. This can be achieved by overriding the header with a custom value:

Reviewer: ```suggestion
In certain scenarios, the `Conversation Id` must be assigned manually in cases where NServiceBus can't infer when messages belong to the same conversation. For example, when a `CancelOrder` message needs to be part of an existing order conversation, then the Order Id can be used for as the Conversation Id. Manually assigning a `Conversation Id` can be achieved by overriding the header with a custom value:
```
